debian/jitsi-meet-electron.buildinfo

```diff
@@ -0,0 +1,163 @@
+Format: 1.0
+Source: __NAME__
+Binary: __NAME__
+Architecture: amd64
+Version: __VERSION__
+Checksums-Md5:
+ __MD5__ __SIZE__ __FILE__
+Checksums-Sha1:
+ __SHA1__ __SIZE__ __FILE__
+Checksums-Sha256:
+ __SHA256__ __SIZE__ __FILE__
+Build-Origin: Ubuntu
+Build-Architecture: amd64
```

Comment:
Out of curiosity, do you how things would look like if we'd want to do arm64 builds?

Reply:
Then the arch should be a parameter, and the post-build step would need to iterate over both debs. I suggest that we do this once we really provide arm64 builds, as that anyway requires some refactorings. But if you think otherwise I can also directly add it.